website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -112,6 +112,65 @@ For Slack and Microsoft Teams notifications, the payload conforms to the respect
 }
 ```
 
+### Workspace Notification Payload
+
+| Name                                         | Type   | Description                                                               |
+| -------------------------------------------- | ------ | ------------------------------------------------------------------------- |
+| `payload_version`                            | number | Currently always "2".                                                     |
+| `notification_configuration_id`              | string | ID of the configuration which defined this notification.                  |
+| `notification_configuration_url`             | string | URL of the Notification Configuration.                                    |
+| `trigger_scope`                              | string | "assessment" for workspace notifications.                                 |
+| `trigger`                                    | string | Value of the trigger which caused the notification to be sent.            |
+| `message`                                    | string | Human-readable reason for the notification.                               |
+| `details`                                    | object | Object containing details specific to the notification.                   |
+| `details.new_assessment_result`              | object | The most recent Assessment Result which triggered the notification.       |
+| `details.new_assessment_result.id`           | string | ID of the Assessment Result.                                              |
+| `details.new_assessment_result.url`          | string | URL of the Assessment Result.                                             |
+| `details.new_assessment_result.succeeded`    | bool   | Whether or not Assessment succeeded.                                      |
+| `details.new_assessment_result.drifted`      | bool   | Whether or not Assessment detected drift.                                 |
+| `details.new_assessment_result.created_at`   | string | Timestamp of the Assessment Result                                        |
+| `details.prior_assessment_result`            | object | The Assessment Result prior to the one which triggered the notification.  |
+| `details.prior_assessment_result.id`         | string | ID of the Assessment Result.                                              |
+| `details.prior_assessment_result.url`        | string | URL of the Assessment Result.                                             |
+| `details.prior_assessment_result.succeeded`  | bool   | Whether or not Assessment succeeded.                                      |
+| `details.prior_assessment_result.drifted`    | bool   | Whether or not Assessment detected drift.                                 |
```

Comment:
This is "prior" --> how is this different from the new one that we have above? I think we should specify that in the description!